tsconfig.eslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["src", "vite.config.ts", ".storybook/**.ts", "tailwind.config.ts"]
```

Comment:
If this still works it's a little simpler

```suggestion
  "include": ["src", "vite.config.ts", ".storybook", "tailwind.config.ts"]
```

Reply:
Unfortunately it does not work
![image](https://github.com/yobgob/too-many-hooks/assets/39521214/6a59c5a2-029d-420a-87c1-882646d1d63e)